specification/attestation/resource-manager/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -341,6 +341,80 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Attestation/defaultProviders": {
```

Comment:
Currently yes, we set them to be tenant level because these resources are regional resources and not limit to customers subscription.
Do you have any recommendation on whether these two Resources (Microsoft.Attestation/defaultProviders and Microsoft.Attestation/locations/defaultProviders) should be tenant level or subscription level?

Reply:
For tenant level API your RP will need to build RBAC checks. For such tenant level calls the recommended way is to go towards "Azure Resource Graph" (ARG) which provides tenant level queryability. Few questions 
1. Will defaultProviders be modeled as proxy resources? 
2, Are they same across tenants?

Reply:
Will defaultProviders be modeled as proxy resources?
> Yes.
>  /providers/Microsoft.Attestation/locations/{location}/defaultProvider
> RoutingType: ProxyOnly, LocationBased 
> 
> /providers/Microsoft.Attestation/defaultProviders
> RoutingType: ProxyOnly, Fanout 

Are they same across tenants?  
> Yes they are same across tenants